### PR TITLE
Hotfix: set SINGLE_ASSET_PAYMENT_CONTRACT_ADDRESS in env file

### PR DESCRIPTION
**Pull request by snario**

### CODE REVIEW

**Comment by snario** [comment]
@kerzhner so after this is merged, should we cherry-pick the commit into a PR into `master`? Or just overwrite `deploy` with some future version of `master` later, ignoring this commit history?

File: .env.production-ropsten [hunk 1]
```diff
@@ -48,7 +48,7 @@ REACT_APP_FIREBASE_URL = $FIREBASE_URL
 REACT_APP_FUNDING_STRATEGY = 'Virtual'
 REACT_APP_HUB_DESTINATION = $HUB_DESTINATION
 REACT_APP_LOG_DESTINATION = $LOG_DESTINATION
-REACT_APP_SINGLE_ASSET_PAYMENT_CONTRACT_ADDRESS = '0xde8585b18e89ad34521f87f236dd745b1c4af99b'
```
Comment (kerzhner) [inline .env.production-ropsten:51]: Nit, let's keep these env vars alphabetized.

**Comment by kerzhner** [comment]
> @kerzhner so after this is merged, should we cherry-pick the commit into a PR into `master`? Or just overwrite `deploy` with some future version of `master` later, ignoring this commit history?

I think it would be best to keep `deploy` a subset of `master` for as long as possible. So I would `--force` push to `master` whenever `master` contains the fix.

**Comment by kerzhner** [comment]
Also, as a side note, our nightly release job is [fairly fragile] (https://github.com/statechannels/monorepo/blob/master/.circleci/config.yml#L419) in that it depends on a seamless rebase of deploy onto master.

**Comment by spersico** [comment]
> Also, as a side note, our nightly release job is [fairly fragile](https://github.com/statechannels/monorepo/blob/master/.circleci/config.yml#L419) in that it depends on a seamless rebase of deploy onto master.

:smile: that's exactly what I did, but I did that manually

**Pull request merged**